### ISSUE TITLE
Unclear instruction for certifier report

### ISSUE THREAD

**Issue by mattgarrish** [w3c/publ-a11y#510]
In the detail conformance reporting section, the instructions for the certifier's report are:

> - display "For more information refer to the certifier's report"
> - with certifier_report as link.

But what does the last bullet mean, as it isn't a "display" instruction:
- the full link is to be displayed after the string? (in which case there should be a space at the end of the statement before)
- the words "certifier's report" in the preceding text are supposed to be linked?
- the whole string from the line before is supposed to be a link?
